app-template/src/types/portal-config.d.ts

```diff
@@ -1,12 +1,14 @@
 /// <reference types="synapse-react-client" />
 
-import { CardContainerLogicProps } from "synapse-react-client/dist/containers/CardContainerLogic";
-import { MarkdownSynapseProps } from "synapse-react-client/dist/containers/MarkdownSynapse";
-import { StackedBarChartProps } from "synapse-react-client/dist/containers/StackedBarChart";
-import { QueryWrapperMenuProps } from "synapse-react-client/dist/containers/QueryWrapperMenu";
-import { QueryBundleRequest } from "synapse-react-client/dist/utils/jsonResponses/Table/QueryBundleRequest";
-import { QueryResultBundle } from "synapse-react-client/dist/utils/jsonResponses/Table/QueryResultBundle";
-import { StackedBarChartControlProps } from "../custom-components/StackedBarChartControl";
+import { CardContainerLogicProps } from 'synapse-react-client/dist/containers/CardContainerLogic'
+import { MarkdownSynapseProps } from 'synapse-react-client/dist/containers/MarkdownSynapse'
+import { StackedBarChartProps } from 'synapse-react-client/dist/containers/StackedBarChart'
+import { QueryWrapperMenuProps } from 'synapse-react-client/dist/containers/QueryWrapperMenu'
+import { QueryBundleRequest } from 'synapse-react-client/dist/utils/jsonResponses/Table/QueryBundleRequest'
+import { QueryResultBundle } from 'synapse-react-client/dist/utils/jsonResponses/Table/QueryResultBundle'
+import { HomeButtonControlProps } from '../portal-components/HomeButtonControlWrapper'
+import { ExploreButtonControlProps } from '../portal-components/ExploreButtonControlWrapper'
+import { QueryWrapperProps } from 'synapse-react-client/dist/containers/QueryWrapper'
```

Comment:
does not exist yet, still needs to get exported from SRC, will break build if staging is run for any portal